teachers_digital_platform/jinja2/teachers_digital_platform/activity_page.html

```diff
@@ -62,6 +62,15 @@ <h3 class="h5">Student materials</h3>
                 </p>
             {% endif %}
     </div>
+    <div>
+        <h2 class="u-mt45">Explore related resources</h2>
+        <ul>
+            {% set parent_page = page.get_parent().url %}
+            <li><a class="a-link a-lnk__icon" href="{{ parent_page }}?q=&topic={{page.get_subtopic_ids() | join('&topic=')}}&grade_level={{ page.get_grade_level_ids() | join('&grade_level=') }}">Search for related CFPB activities</a></li>
```

Comment:
```suggestion
            <li><a class="a-link a-lnk__icon" href="{{ parent_page }}?q=&topic={{ page.get_subtopic_ids() | join('&topic=') }}&grade_level={{ page.get_grade_level_ids() | join('&grade_level=') }}">Search for related CFPB activities</a></li>
```

Might be better to put this as a method on the page class, e.g. `page.get_related_topics_url()`, which would allow for writing unit tests against this.

Also should this be `a-link__icon` not `a-lnk__icon`?

Reply:
I've moved this into a method on the Activity page class instead